cpp/src/reductions/scan/scan.cuh

```diff
@@ -33,6 +32,11 @@ rmm::device_buffer mask_scan(column_view const& input_view,
                              rmm::cuda_stream_view stream,
                              rmm::mr::device_memory_resource* mr);
 
+std::unique_ptr<column> ewm(column_view const& input,
```

Comment:
Why is this named `ewm` when everything else is named `ewma`?

Reply:
Ah, this originally dispatched to all the other weighted moments (`ewmvar` and `ewmstd`) which I removed to reduce the complexity here. Would it be better to just remove it for now and then put it back in when those functions come along?